### PR TITLE
Bug 561435 API revision: requirements resolution: test coverage

### PR DESCRIPTION
**Pull request by eparovyshnaya**
1. tests for Bundle info suppliers
 2. special resource bundle `data.requirements`
 3. tests for `BundleRequirements`
 4. tests for `LicensingFeatureCapabilitiesFromBundleTest`

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/.settings/.api_filters [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests.data.requirements/.settings/.api_filters]: this file should be removed

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests.data.requirements/plugin.xml]: Is this needed?

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Contributors:
+#     ArSysOp - initial API and implementation
+###############################################################################
+
+source.. = src/
+output.. = bin/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               plugin.xml
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties:19]: is this needed?

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleNameTest.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleNameTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleRequirementsTest.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleRequirementsTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendorTest.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendorTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/ComponentRequirementsTest.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/ComponentRequirementsTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/DataBundle.java [hunk 8]
```diff
@@ -0,0 +1,15 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/DataBundle.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/LicensingFeatureCapabilitiesFromBundleTest.java [hunk 9]
```diff
@@ -0,0 +1,34 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/LicensingFeatureCapabilitiesFromBundleTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementsFromCapabilityTest.java [hunk 10]
```diff
@@ -0,0 +1,13 @@
+package org.eclipse.passage.lic.internal.equinox.requirements;
```
Comment (ruspl-afed) [inline tests/org.eclipse.passage.lic.equinox.tests/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementsFromCapabilityTest.java:13]: header missed

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties (hunk 3 above)
Comment (ruspl-afed) [reply tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties]: needs to be removed

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/.settings/.api_filters (hunk 1 above)
Comment (eparovyshnaya) [reply tests/org.eclipse.passage.lic.equinox.tests.data.requirements/.settings/.api_filters]: Done

File: tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties (hunk 3 above)
Comment (eparovyshnaya) [reply tests/org.eclipse.passage.lic.equinox.tests.data.requirements/build.properties]: Removed

**Pull request merged**